### ISSUE TITLE
Analytics Implementation/Support for Authenticate experience (Profile) -- Bad address indicator email

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#42197]
# Google Analytics Implementation or QA Request

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [X] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**

This request is for the authenticated experience team for the VA.gov profile. We are launching a feature called the bad address indicator, which lets a veteran know that the VA has sent them mail that's been returned to the VA, which leads us to believe that the VA does not have the correct address on file for the veteran. To support this feature, we are piloting an email (in partnership with our backend team, VA Profile), and we'd like to set up a funnel report in GA to track the following:

- How many people click the email link to update their mailing address
- How many people click the email link AND complete login
- How many people click the email link, login, and then update their mailing address
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
- [Here is the link to the email we plan to launch in the VANotify staging environment](https://staging.notifications.va.gov/service/5e5cded3-3c76-46a3-9eef-cb63589e76be/templates/149f000b-3cde-4150-b6aa-ce47da8e124b)
- We already have tracking on mailing address updates, but the link for where people can do that will be different by the time this feature launches. Currently, people can update their mailing address at http://www.va.gov/profile/personal-information. By the time this feature launches, mailing address updates will have moved to http://www.va.gov/profile/contact-information.
    
- **When is the launch date?**
    - Launch date for the email is TBD, but likely within the next few weeks

- **What interactions need to be tracked?** 
    
This is a funnel, and the following needs to be tracked:
    
- How many people click the email link to update their mailing address. The link we are going to use for this will be https://va.gov/profile/contact-information](https://va.gov/profile/contact-information/?utm_source=VANotify&utm_medium=email&utm_campaign=bad-addr-indicator
- How many people click the email link AND complete login. If someone clicks the link and completes login, they will land in the profile at https://va.gov/profile/contact-information/?postLogin=true
- How many people click the email link, login, and then update their mailing address. After someone clicks the link, logs in, and lands in the profile, they can update their mailing address right from https://va.gov/profile/contact-information/?postLogin=true if they choose. We already have tracking on mailing address updates.

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - All staging users have access to this section of the profile

- **Any additional comments**

    - N/A

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by Samara-Strauss** [comment]
cc/ @SKasimow for awareness that I opened this ticket

**Comment by Samara-Strauss** [comment]
> Would you want to get more granular within the funnel? ie; the mailing address update requires a few clicks before the final form submission. Would you and your team want to track the clicks to update the address and the actual submission? 

Yes, this would be great! I wasn't sure if we could get this level of detail, but we'd love that.

The report looks great, and we'll check if data is populating accordingly once the email starts going out (may still be a few weeks off).

**Comment by jonathan-epstein13** [comment]
Great @Samara-Strauss ! Will keep you posted once it is implemented.

**Comment by jonathan-epstein13** [comment]
@Samara-Strauss We've been able to add the extra step in the funnel report as requested. Please let us know if you need anything else for this report. The funnel report can be found in the All VA.gov Website Data [Production] property under the Explore tab. I had to make a duplicate due to the editing of the steps.

**Comment by jonathan-epstein13** [comment]
@Samara-Strauss We've been able to add the extra step in the funnel report as requested. Please let us know if you need anything else for this report. The funnel report can be found in the All VA.gov Website Data [Production] property under the Explore tab. I had to make a duplicate due to the editing of the steps.

**Comment by Samara-Strauss** [comment]
Thank you, this looks great! I think we can go ahead and delete the other version of the report (it looks like Jenna might need to do it). Otherwise, this ticket should b e good to close.

![image](https://user-images.githubusercontent.com/34068740/174147612-c439a552-9fa6-4d2a-a67c-fbf8c006a34a.png)

**Comment by michelle-dooley** [comment]
Great, thanks @jennaouellet!!  I will go ahead and close this ticket.  @Samara-Strauss if you have any questions or need anything else please let us know.
